VideoPlayer/VideoPlayer.js

```diff
@@ -84,7 +85,7 @@ const RootContainer = SpotlightContainerDecorator(
 
 const ControlsContainer = SpotlightContainerDecorator(
 	{
-		enterTo: '',
+		enterTo: 'default-element',
```

Comment:
Move focus to "the slider" from "the back button".

Reply:
🆗

Reply:
🆗

Reply:
For the record, you can check the behavior by entering focus via 5way from the back button to controls.